sklearnex/preview/decomposition/tests/test_incremental_pca.py

```diff
@@ -200,7 +200,7 @@ def test_sklearnex_partial_fit_on_gold_data(dataframe, queue, whiten, num_blocks
 
     X_df = _convert_to_dataframe(X, sycl_queue=queue, target_df=dataframe)
     transformed_data = incpca.transform(X_df)
-    check_pca_on_gold_data(incpca, dtype, whiten, transformed_data)
+    check_pca_on_gold_data(incpca, dtype, whiten, _as_numpy(transformed_data))
```

Comment:
similar comment here, it's better to use dtype arg from check_pca_on_gold_data in the function for tolerance setting, it allows to avoid additional _as_numpy call

Reply:
_as_numpy call already exists inside the method, only tolerance setting is necessary to be updated using dtype arg